bin/run-cloud-deployment

```diff
@@ -0,0 +1,16 @@
+#! /usr/bin/env bash
+
+# DOC: Run deploy scripts from Docker.
+
+# Usage:
+# bin/build-cloud
+# bin/run-cloud <command> <tag> <config_file>
+
+export COMMAND="${1:-deploy}"
+export IMAGE_TAG="${2:-latest}"
+export CONFIG="${3:-cloud/deploys/dev_aws/civiform_config.sh}"
+export AWS_ACCESS_KEY_ID=$(aws --profile default configure get aws_access_key_id)
+export AWS_SECRET_ACCESS_KEY=$(aws --profile default configure get aws_secret_access_key)
+echo "Running ${COMMAND} ${IMAGE_TAG} ${CONFIG}"
+docker compose -f cloud/docker-compose.yaml up
+docker compose -f cloud/docker-compose.yaml rm -fsv
```

Comment:
question (blocking): what do folks think of moving these scripts into `cloud/shared/bin`?